CHANGELOG.md

```diff
@@ -48,6 +48,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+- [2014](https://github.com/umee-network/umee/pull/2014) Fix reward distribution when zero bonded uTokens.
```

Comment:
probably we don't need that - because it's a fix to a not released code. Otherwise should be moved to `Fixes` section.

Reply:
got it